src/scripts.js

```diff
@@ -168,10 +172,14 @@ function searchHomeRecipeByTag() {
     const tagSelected = determineSelectedTagValue()
     const tagSelectedList = recipeRepository.filterTag(tagSelected)
     allRecipes.innerHTML = ''
-    if (tagSelected === 'reset all') {
+    if (tagSelected === 'reset all' && homeView) {
```

Comment:
We will have a merge conflict with this. I was able to filter by radio button but I had to change some things so this will be something to address when we merge. I added you as a reviewer so you will see what I'm talking about